recipes/python-bioext/activate.sh

```diff
@@ -0,0 +1,2 @@
+export
```

Comment:
```suggestion
```

Reply:
Pezzo di me... I thought I removed all the print debugs, thanks again, signore :eagle::eye: